modules/exploits/windows/fileformat/allplayer_m3u_bof.rb

```diff
@@ -28,6 +28,7 @@ def initialize(info = {})
         ],
       'References'     =>
         [
+          [ 'CVE', '2013-7409' ],
```

Comment:
The CVE description mentions 5.8.1 but the module description mentions 2.8.1.
Fix module description.